common/client_config.py

```diff
@@ -46,3 +44,26 @@ class ClientConfig(BaseModel):
 
     # How long to wait before considering a server as disconnected.
     server_timeout_seconds: float = 2.0
+
+    # Game mode, "competitive" or "local_evaluation"
+    game_mode: str = "competitive"
+
+    # Competitive agent configuration, change the sciper to yours, and the name to
+    # your "train name". Simply modify the path to the agent file you want to use. 
+    competitive_agent: dict = {
```

Comment:
This should be dict of a specific type. Otherwise, random fields will get deserialized.